tests/bootstrap_tests/test_resolver.py

```diff
@@ -339,156 +377,118 @@ def find_mock_impl(*args, **kwargs):
 
 
 class TestResolverPriority(TestResolverBase):
+    """
+    This test ensures that the following priority is respected when multiple pipeline configurations
+    are found.
 
-    @patch("tank_vendor.shotgun_api3.lib.mockgun.Shotgun.find")
```

Comment:
All tests that tested the priority of pipeline configurations over others have been rewritten for thoroughness and brevity.